### PR TITLE
Davide/lumi 69 fix suggestion

### PR DESCRIPTION
**Pull request by aittalam**
## I already...

- [ ] added some tests for any new functionality;
- [ ] updated the documentation.

### CODE REVIEW

File: lumigator/python/mzai/evaluator/configs/jobs/common.py [hunk 1]
```diff
@@ -3,13 +3,12 @@
...
 from pydantic import Field
 from pydantic_yaml import parse_yaml_file_as, to_yaml_file
 
-from configs.common import EvaluatorConfig
-from configs.wandb import WandbRunConfig
-from paths import AssetPath, PathPrefix
-
 
 class JobConfig(EvaluatorConfig):
     """Configuration that comprises the entire input to an LM Buddy job.
```
Comment (veekaybee) [inline lumigator/python/mzai/evaluator/configs/jobs/common.py:14]:
```suggestion
    """Configuration that comprises the entire input to an evaluation job.
```

**Comment by veekaybee** [comment]
Tested and working locally, thank you! 🚀

**Review by veekaybee — COMMENTED** [review]
we may need to change the LD_PRELOAD hash: https://github.com/mozilla-ai/lumigator/compare/LUMI-69-fix?expand=1#diff-ce78120cb71224fbf5e4ba2d2d2ae2e2122e56903af3a094a3f0396ece4d8544R61

What do you think of adding a few logs at the submission statement from the Lumigator side for visiblity? 

https://github.com/mozilla-ai/lumigator/compare/LUMI-69-fix?expand=1#diff-27713baa81cbb6a09b2a240dbf343b1fb3916712022f5a5c224e2907949855d5R149

**Comment by aittalam** [comment]
> we may need to change the LD_PRELOAD hash: https://github.com/mozilla-ai/lumigator/compare/LUMI-69-fix?expand=1#diff-ce78120cb71224fbf5e4ba2d2d2ae2e2122e56903af3a094a3f0396ece4d8544R61
> 
> What do you think of adding a few logs at the submission statement from the Lumigator side for visiblity?
> 
> https://github.com/mozilla-ai/lumigator/compare/LUMI-69-fix?expand=1#diff-27713baa81cbb6a09b2a240dbf343b1fb3916712022f5a5c224e2907949855d5R149

Re: the hash, it weirdly seems to be still working - perhaps moving from the list of packages to the requirements.txt file did not change it because it is still the same list of libs?

I agree about the logs! Possibly related to this, I see there are some conflicts here and I want to be mindful about your preexisting code, so: what would work best for you? I am happy to hand this PR out to you for merging, but also to deal with the conflicts myself it that's a bother

**Comment by veekaybee** [comment]
Sorry, I missed that this was against my branch rather than main, feel free to merge there directly!

**Pull request merged**